src/pages/home/HomePage.tsx

```diff
@@ -16,6 +16,7 @@ const HomePage = () => {
           {availableItems.map((item) => (
             <Grid key={item.title} item width="25%">
               <Card
+                id="cardComponent"
```

Comment:
@leleOFA questo id non ha senso, le card sono ripetute

Reply:
fix it